sktm/jenkins.py

```diff
@@ -82,13 +86,54 @@ def __get_build(self, job, buildid):
                 return build
             except Exception as e:
                 logging.warning("catch %s: %s" % (type(e), e))
-                logging.info("now sleep 60s and try again")
-                time.sleep(60)
+                logging.info("now sleep %ds and try again" % interval)
+                time.sleep(interval)
 
         logging.error("fail to get build after retry %d times" %
                       self.retry_cnt)
         raise e
 
+    def __get_job_prop(self, job, attr, interval):
+        """
+        Get property of Jenkins job. Retry Jenkins self.retry_cnt times
+        in case of temporary network failures.
+
+        Args:
+            job:        Jenkins job.
+            attr:       Attribute of job.
```

Comment:
The attribute has to be callable so there was no need to change the wording from `method` as it was previously :)

Reply:
Ok, roll back to use 'method' :-)